examples/features/authentication/pushed-authorization-request/.env.example

```diff
@@ -0,0 +1,6 @@
+DOMAIN=
```

Comment:
Should we add comments explaining what each config is and where to get it/generate it (if applicable)?

Reply:
Good call out, I've updated it with guidance comments 👍